src/Tribe/Main.php

```diff
@@ -961,6 +961,8 @@ public function init() {
 			$this->maybeSetTECVersion();
 
 			$this->run_scheduler();
+
+			$a = [ 1 ];
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: eb72efbc17be3ede5cf2c9302bbcf508</sub>